src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -79,20 +54,240 @@ Developers can use CHERI to build fine-grained spatial and temporal memory
+// Therefore, the capability level is adjusted using the <<ACPERM>> instruction (see <<section_cap_level_change>>) and are queried using <<GCPERM>>.
+But unlike architectural permissions, CL can be reduced even if the capability is sealed.
+
+NOTE: A capability with <<section_cap_level,CL>>=1 is referred to as _global_ and with <<section_cap_level,CL>>=0 as _local_.
+
+.{cheri_1levels_ext_name} summary table for stored capabilities
+[#cap_level_store_summary,width="100%",options=header,halign=center,cols="1,1,1,1,5"]
+|==============================================================================
+   3+|Auth cap field       | Data cap field |
+    h|*W*    h|*C* h|*SL* h|*CL* h| Notes
+.3+.^|1  .3+.^| 1   | 1    | X    | Store data capability unmodified
+               .2+.^| 0    | 1    | Store data capability unmodified (`CL ≥ ~SL`)
+                           | 0    | Store data capability with validity tag cleared (`CL < ~SL`)
+|==============================================================================
+
+NOTE: if W=0 or C=0 then SL is irrelevant
+
+.{cheri_1levels_ext_name} additional rules for loading capabilities
+[#cap_level_load_summary,width="100%",options=header,align=center,cols="1,1,1,1,1,1,6"]
+|==============================================================================
+   4+|Auth cap field                  2+| Data cap field |
+    h|*R*   h|*C* h|*EL*      h|*CL*   h| Tag h| Sealed h|Action
+.2+.^|1 .2+.^| 1 .2+.^| 0 .2+.^| X .2+.^| 1    | Yes     |Load data capability with `CL=min(auth.CL, data.CL)`, EL unchanged
+                                               | No      |Load data capability with `EL=0, CL=min(auth.CL, data.CL)`
+   6+| All other cases                                   |Load data capability with EL, CL unmodified
+|==============================================================================
+
+NOTE: The current specification only defines up to two levels, equivalent to _local_ and _global_ capabilities from CHERI v9, Morello and CHERIoT.
+Future extensions are permitted to add more levels.
+
+[#sec_cap_sdp]
+==== Software-Defined Permissions (SDP)
+The metadata also contains an encoding-dependent number of software-defined permission (SDP) bits.
+They can be inspected by the kernel or application programs to enforce restrictions on API calls (e.g. permit/deny system calls, memory allocation, etc.), but are not interpreted by the CPU otherwise.
+
+NOTE: While these bits are not interpreted by the hardware, modification follows the same rules as architectural permissions: SDP bits can only be cleared and never set.
+This property is required to ensure restricted programs cannot forge capabilities that would pass the software-enforced checks.
+
+[#section_special_caps]
+==== Special Capabilities
+
+[#infinite-cap,reftext="Infinite"]
+===== Infinite Capability
+
+The _Infinite_ capability grants all permissions while its bounds also cover the whole address space.
```

Comment:
Not sure what we discussed about this, but there are conceivable systems where there is no infinite capability like CHERIoT. Should we mark the existence of this infinite capability as optional?

Reply:
I agree that it should be optional, but not sure this should be mentioned here. Maybe in the appendix?

Reply:
Added note in appendix.